build/config/webpack.base.js

```diff
@@ -30,5 +30,7 @@ export default {
     new webpack.OldWatchingPlugin(),
     new webpack.NoErrorsPlugin(),
   ],
-  externals: [],
+  externals: [
+    'dtrace-provider',
```

Comment:
Bunyan uses dtrace-provider to do some special logging on OSX. By marking this as external and then not including it in the app node_modules it fails to resolve when bunyan requires it, which bunyan silently ignores. The dtrace-provider module is broken in recent electron versions due to a complex ABI issue.